pkg/k8shandler/collection.go

```diff
@@ -108,26 +113,29 @@ func (clusterRequest *ClusterLoggingRequest) CreateOrUpdateCollection() (err err
 			return
 		}
 
-		if err = clusterRequest.removeFluentd(); err != nil {
+		if err = clusterRequest.removeCollector(constants.FluentdName); err != nil {
```

Comment:
related to previous comments about changing something like  `constants.FluentdName -> constants.collectorName`